_posts/de/newsletters/2025-05-02-newsletter.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Bitcoin Optech Newsletter #352'
+permalink: /de/newsletters/2025/05/02/
+name: 2025-05-02-newsletter-de
+slug: 2025-05-02-newsletter-de
+type: newsletter
+layout: newsletter
+lang: de
+---
+Der Newsletter dieser Woche verweist auf Vergleiche verschiedener Cluster-Linearisationstechniken und fasst kurz die Diskussion über die Erhöhung oder Abschaffung des `OP_RETURN`-Größenlimits in Bitcoin Core zusammen. Ebenfalls enthalten sind unsere regulären Abschnitte mit Ankündigungen neuer Releases und Release-Kandidaten sowie Zusammenfassungen wichtiger Änderungen an populärer Bitcoin-Infrastruktur.
+
+## Nachrichten
+
+- **Vergleich von Cluster-Linearisationstechniken:**
+  Pieter Wuille [veröffentlichte][wuille clustrade] auf Delving Bitcoin einige grundlegende Abwägungen zwischen drei verschiedenen Cluster-Linearisationstechniken und ergänzte dies durch [Benchmarks][wuille clusbench] der jeweiligen Implementierungen. Mehrere andere Entwickler diskutierten die Ergebnisse und stellten Rückfragen, auf die Wuille antwortete. {% assign timestamp="0:41" %}
```

Comment:
Just glanced at this again, and noticed that this translation contains timestamps, but they don’t seem to appear in the German rendering either way. I think it would be better to remove the timestamps, in case we some day also get people do a German Recap. ;)